### ISSUE TITLE
Mistaken Solr query as SQL query

### ISSUE THREAD

**Issue by mcgitty** [zaproxy/zaproxy#2158]
Apache Solr search syntax (superset of lucene) allows keywords: AND, OR, NOT:

  http://wiki.apache.org/solr/SolrQuerySyntax
  http://lucene.apache.org/core/3_6_0/queryparsersyntax.html#Boolean%20operators

Zaproxy reports this as a high vulnerability of SQL Injection. See attached alert detail copy:
[zaproxy-bug.pdf](https://github.com/zaproxy/zaproxy/files/75083/zaproxy-bug.pdf)

**Comment by kingthorin** [comment]
I did run the pdf through [virustotal](https://www.virustotal.com/gui/url/6adebb41b9a4244344b162cd31acda484bb7ecc528a763ebed74b5ccc34bc82a/detection), however, so that others don't have to open it.

![image](https://user-images.githubusercontent.com/7570458/72008720-774c7e00-3222-11ea-996e-d1223e2c6fb8.png)

**Comment by kingthorin** [comment]
Agreed this is a false positive, however it is likely to be true of many search components. Of course analysts have to be able to test/confirm findings. You can use [Alert Filters ](https://github.com/zaproxy/zap-extensions/wiki/HelpAddonsAlertFiltersAlertFilter) to work around this for repeated scans.
